train_dolly.py

```diff
@@ -91,6 +91,7 @@
 dbutils.widgets.text("local_training_root", "", "local_training_root")
 dbutils.widgets.text("dbfs_output_root", "", "dbfs_output_root")
 dbutils.widgets.text("experiment_id", "", "experiment_id")
+dbutils.widgets.combobox("gpu_family", "a100", ["v100", "a100"])
```

Comment:
I'm a little wary of this because at this point I have not successfully made things work on A10s. I was getting some other error that I have not yet been able to diagnose. I wanted to get the V100 config changes in, and then maybe we could get a working A10 config in a subsequent PR

Reply:
I believe I've got the A10 config working now...